src/Components/Table/Context/TableColumnsContext.tsx

```diff
@@ -201,7 +208,7 @@ export const TableColumnContextProvider = ({
         filteredColumns,
         columns,
         setColumns: handleSetColumns,
-        visible: visible,
+        visible: isColumnManagementActive,
         setVisible: handleSetVisible,
         clearSelectedLine: handleClearSelectedLine,
```

Comment:
This is passing a completely new object on any re-render, maybe this is causing the re-render I just noticed today? I wouldn't be surprised.